test_scripts/Resumption/InteriorVehicleData/005_InteriorVD_resumption_all_modules_with_moduleId_unexpected_disconnect.lua

```diff
@@ -0,0 +1,57 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: Successful resuming of interior vehicle data after transport disconnect
+--  in case app was subscribed to some of modules (one of each module type)
+--
+-- Precondition:
+-- 1. HMI and SDL are started
+-- 2. Mobile app with REMOTE_CONTROL hmi type is registered and activated
+-- 3. App is subscribed to all modules via GetInteriorVehicleData(moduleType, moduleId)
+--
+-- Sequence:
+-- 1. Transport disconnect and reconnect are performed
+-- 2. App starts registration with actual hashId after unexpected disconnect
+-- SDL does:
+-- - a. send RC.GetInteriorVehicleData(subscribe=true, moduleType,moduleId) to HMI
+--    during resumption data for each previously subscribed module
+-- - b. respond RAI(SUCCESS) to mobile app
```

Comment:
please add SDL get all successful responses for request with isSubscribed=true before SDL respond RAI(success=true, result code = SUCCESS) to mobile

Reply:
Done